src/odemis/driver/piezomotor.py

```diff
@@ -532,21 +577,22 @@ def getIndexStatus(self, axis):
                 indexed = False
             return mode, position, logged, indexed
         except Exception as ex:
-            logging.error("Failed to parse index status %s: %s" % (ret, ex))
+            logging.exception("Failed to parse index status %s: %s" % (ret, ex))
```

Comment:
`logging.exception()` automatically logs the exception, so you don't need to put it in the message. 
BTW, it's recommended to pass the variables as arguments when using `logging`, though it doesn't really matter.